src/sphinx/extensions/index.rst

```diff
@@ -17,7 +17,7 @@ Official Extensions
 Third Parties
 =============
 
-* `Gradle plugin <https://github.com/alphagov/gradle-gatling-plugin>`_
+* `Gradle plugin <https://github.com/lkishalmi/gradle-gatling-plugin>`_
```

Comment:
Forget my previous comment. It seems @alxndrsn got a new hair style, moved to a different country and is not longer involved. Damn, the guy owes a dinner!